demo-template/app/index.css

```diff
@@ -0,0 +1,68 @@
+:root {
```

Comment:
Why is the app AND the template here? I guess the app is a materalized form of the actual template right?

Reply:
It made it easier for me to build this having a working version without the tokens.